lib/marathon_client.ex

```diff
@@ -39,11 +39,16 @@ defmodule MarathonEventExporter.MarathonClient do
       {:ok, {r, ssep}}
     end
 
-    def handle_info(%HTTPoison.AsyncChunk{chunk: chunk}, {_, ssep}=state) do
+    def handle_info(msg=%HTTPoison.AsyncChunk{chunk: chunk}, {_, ssep}=state) do
```

Comment:
I'm confused about which side of the `=` what is... 🤔

Reply:
It's a pattern match, and I believe you can have as many `=` as you like:
```iex(1)> a = {b, c} = {3, 5}; {a, b, c}
{{3, 5}, 3, 5}
```

The `msg=%HTTPoison.AsyncChunk{chunk: chunk}` is matching the arg value against both `msg` (which captures the whole struct) and `%HTTPoison.AsyncChunk{chunk: chunk}` (which extracts just the chunk) so that the body of the function can use both `msg` and `chunk`.

Reply:
So... `%HTTPoison.AsyncChunk{chunk: chunk}=msg` would do the same thing?

Reply:
Yes.

Reply:
Do we want to be consistent? In the same line you have `{_, ssep}=state` which is the other war around.

Reply:
Consistency is good. I'm probably going to remove most of that debug logging soon, but I'll fix the ordering for now.